vipps-qr-api.md

```diff
@@ -313,3 +313,105 @@ does what one might expect, it deletes the QR. Once deleted, merchants can gener
 In addition to the `DELETE`-endpoint, it is also possible to add a `ttl`-attribute in the original `POST`-request. This attribute sets how many seconds the QR will live, before it is deleted permanently.
 
 Tip: If you want the same QR in different formats, perform `GET` calls on the same `id` with different `accept` headers and test what works best.
+
+## Merchant Callback QR codes
+
```

Comment:
Add a "NB - this is not ready in production yet, estimated release is Q2 2023"

Reply:
+1